rest/auth.py

```diff
@@ -311,6 +305,37 @@ def inner(*args, **kwargs):
 
     return decorator
 
+
+# Retrive current user role and login
+def get_current_user():
```

Comment:
Please add docstring to explain what this function does, notably in which cases it aborts.